frontends/mit-open/webpack.config.js

```diff
@@ -66,7 +93,7 @@ module.exports = (env, argv) => {
   const config = {
     mode,
     context: __dirname,
-    devtool: "source-map",
+    devtool: isProduction ? "source-map" : "eval-source-map",
```

Comment:
`source-map` generates source map files. That is what should be used in production.

We've been getting—sometimes—crappy stacktraces locally. For example, here's one @rhysyngsun got awhile ago:
```
Warning: Failed prop type: Invalid prop `children` supplied to `ForwardRef(Container)`, expected a ReactNode.
Container@http://open.odl.local:8063/root.js:70123:32
../../node_modules/@emotion/react/dist/emotion-element-43c6fea0.browser.esm.js/withEmotionCache/<@http://open.odl.local:8063/root.js:6546:66
OnboardingPage@http://open.odl.local:8063/root.js:242134:89
RenderedRoute@http://open.odl.local:8063/root.js:201393:7
RenderErrorBoundary@http://open.odl.local:8063/root.js:201337:5
DataRoutes@http://open.odl.local:8063/root.js:199962:7
Router@http://open.odl.local:8063/root.js:202021:7
RouterProvider@http://open.odl.local:8063/root.js:199749:7
InnerContextProvider@http://open.odl.local:8063/root.js:5565:20
Provider@http://open.odl.local:8063/root.js:5574:20
_HelmetProvider@http://open.odl.local:8063/root.js:194369:5
QueryClientProvider@http://open.odl.local:8063/root.js:281681:29
RtlProvider@http://open.odl.local:8063/root.js:70256:9
ThemeProvider@http://open.odl.local:8063/root.js:69088:7
ThemeProvider@http://open.odl.local:8063/root.js:70547:7
ThemeProvider@http://open.odl.local:8063/root.js:67213:9
ThemeProvider@http://open.odl.local:8063/root.js:249061:20
AppProviders@http://open.odl.local:8063/root.js:231877:18
```

I've had trouble reproducing this locally at the moment, but I've read that `source-map` doesn't work great with hot module reloading. `eval-source-map` embeds the sourcemaps in the JS file and is supposed to be more reliable with HMR.